packages/analytics-js/src/components/pluginsManager/PluginsManager.ts

```diff
@@ -96,7 +96,7 @@ class PluginsManager implements IPluginsManager {
     // Error reporting related plugins
     if (!state.reporting.isErrorReportingEnabled.value) {
       pluginsToLoadFromConfig = pluginsToLoadFromConfig.filter(
-        pluginName => !(pluginName === 'ErrorReporting'),
```

Comment:
Did you not implement the deprecated plugin functionality yet?

Reply:
Addressed